.github/workflows/docs.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: vale docs/
 
       - name: Install Ruby
-        uses: ruby/setup-ruby@2a7b30092b0caf9c046252510f9273b4875f3db9 # v1.254.0
+        uses: ruby/setup-ruby@472790540115ce5bd69d399a020189a8c87d641f # v1.247.0
```

Comment:
Whait, why is this a downgrade?

Reply:
I'm a bit confused here, because when I checked yesterday it looked like `setup-ruby` was v1.254.0 across all repos. All of these downgrade PR's have been to v1.247.0. I'm going to bump the 3 instances of v1.247.0, verify we're up to date, and then we need to watch these config syncs for a while.